packages/create-instantsearch-app/src/templates/JavaScript Client/src/app.js.hbs

```diff
@@ -17,6 +17,9 @@ function renderHits(query) {
               `<li class="ais-hits--item">
                 <article>
                 {{#if attributesToDisplay}}
+                  {{#if imageAttribute}}
+                  <img src="${ hit.{{imageAttribute}} }" alt="${ hit.{{attributesToDisplay.[0]}} }" />
```

Comment:
I have to put whitespace between curly braces here and at other places else handlebars compilation will fail.
It's fine as the generated code will be formatted anyway

Reply:
I wonder if we could use a different character than `{`, but not sure if it's worth the confusing